cli/src/api/mod.rs

```diff
@@ -114,6 +116,16 @@ impl PhylumApi {
         path: U,
         body: Option<B>,
     ) -> Result<T> {
+        let body = self.send_request_raw(method, path, body).await?;
+        serde_json::from_str::<T>(&body).map_err(|e| PhylumApiError::Other(e.into()))
+    }
+
+    async fn send_request_raw<B: Serialize, U: IntoUrl>(
+        &self,
+        method: Method,
+        path: U,
+        body: Option<B>,
+    ) -> Result<String> {
```

Comment:
If we're going to introduce this function, we should use it for `delete_project()` as well (which is currently using [`IgnoredAny`](https://docs.rs/serde/1.0.147/serde/de/struct.IgnoredAny.html) to throw away the output)

Reply:
Fyi `IgnoredAny` works if we want to ignore the output, but will not work if the output is empty. That's why I handled it this way. Ideally I'd rather not have to do it, but serde cannot handle deserializing an empty string to anything.